### DISCUSSION TITLE
No progress bar in "Tasks" until page reload

### DISCUSSION THREAD

**Discussion by Roycinger** [stashapp/stash#4550] [Q&A] [answered]
**Describe the bug**
When adding new scenes, after pressing "Scan", there is no progress bar until I refresh the page. Also the progress bar won't move until the page is refreshed again.

**To Reproduce**
1. Go to "gear icon" in the top right corner
2. Under Tasks -> Library, press "Scan" button

**Expected behavior**
Updated progress bar and filesnames in Task Queue that are being worked on

**Stash Version: (from Settings -> About):**
v0.18.0

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Chhrome
 - Version 108.0.5359.125

**Additional context**
Stashapp runs in a Docker container on a Synology DS415+. Mounts and environment variables according to the Wiki page (https://github.com/stashapp/stash/wiki/Installing-on-Synology-NAS)

**Comment by ltgorman** [comment]
This is not just a docker issue.  I run stash on a separate PC.  When I have the tasks open in a browser on that machine, the progress bar is live.  When I have it open on another machine the progress bar is, more often than not, not live.  I have very occasionally seen it work on my other machine, however I am not sure why it worked briefly.

**Comment by LeGrosFromage** [comment]
I'm also running in a Docker on Synology 1621+, when I do a Scan the progress bar works correctly each and every time.  I don't recall ever seeing it not work correctly, to be honest.

Desktop system is Linux running Brave browser.

**Comment by WithoutPants** [comment, marked as answer]
This indicates that the websockets connection hasn't been made. This is usually due to a reverse proxy swallowing the requests. More info can be gained by checking the browser developer window -> network tab, looking for a `ws` `/graphql` request.

**Comment by AdultSun** [comment]
I'm not sure what causes this exactly and I can't speak to any of the technical details, but this was happening to me a long time ago too. I also run Stash in a Docker from a separate machine, though I'm not sure if that's relevant. For me it was entirely dependent on the browser I used. At first I was using Brave and the progress bar would consistently fail to update in the same way you describe. I never figured out a way to get it to work besides refreshing the page. Eventually I switched to a Chromium build instead and since then it's worked for me every time.
